send_telegram.py

```diff
@@ -0,0 +1,40 @@
+import requests
+import os
+from logger import logger
+from load_config import *
+
+config = load_config('config.yml')
+
+send_success = False
+
+if "BOT_TOKEN" in config['TELEGRAM'] and "BOT_CHAT_ID" in config['TELEGRAM']:
+   bot_token = config['TELEGRAM']['BOT_TOKEN']
+   bot_chatID = str(config['TELEGRAM']['BOT_CHAT_ID'])
+   send_success = True
+else:
+   logger.info('Telegram not configured -> disabled')
+
+def send_telegram(message):
+
+   global send_success
+
+   if send_success:
+
+      for var in [bot_token, bot_chatID]:
+         if var is None:
+            return
+
+      send_text = 'https://api.telegram.org/bot' + bot_token + '/sendMessage?chat_id=' + bot_chatID + '&parse_mode=Markdown&text=' + message
+      logger.info('TELEGRAM sending '+message)
+      
+      response = requests.get(send_text)
+      logger.debug('TELEGRAM '+str(response.json()))
+
+      if response.status_code != 200:
```

Comment:
There are many cases where it only fails once or a couple times and then works again (i.e. all the 500's and probably some 400's). Not sure if we want to completely disable the whole bot just because it didn't work once.

Reply:
true, i changed to log error, disabling is now only depending on config